articles/django-rest-api-mongodb-notes-app/index.md

```diff
@@ -0,0 +1,226 @@
+# Build a Notes App API with Django REST Framework and MongoDB
+
+### Introduction
+MongoDB is a NoSQL database system. It stores data in JSON-like format instead of the traditional row-column format. The Django framework is a Python framework for web development. The Django REST framework is a powerful toolkit for building extensible APIs. 
+
+In this tutorial, you will learn how to use the powerful Django REST framework and MongoDB in your web projects. You will learn how to put in place CRUD features in a notes app API.
+
+By the end of this tutorial, you will be able to:
+
+- Integrate MongoDB into Django projects.
+- Set up and run a Django REST API.
+- Create CRUD features for a Django REST API.
+- Build a notes app API.
+
+### Prerequisites
+- Python 3 installed on your machine. You can install Python 3 from the [Python website](https://www.python.org/download/releases/3.0/).
+- Some knowledge of Python and Django.
+
+### Setting up a MongoDB server
+Download and install the MongoDB database server from the [official MongoDB website](https://www.mongodb.com/try/download/community).
+
```

Comment:
Installing MongoDB is not always easy for beginners and Linux users. Provide a link to tutorials to guide the installations.

Also provide instructions on how to create the database used for this tutorial.